package.json

```diff
@@ -72,9 +72,7 @@
     "url": "http://github.com/jquery/globalize/issues"
   },
   "dependencies": {
-    "cldrjs": "0.4.3"
-  },
-  "peerDependencies": {
+    "cldrjs": "0.4.3",
```

Comment:
This seems like the wrong fix. We want to keep the peerDependency, to allow users to install the version of their choice. We also want to install it when testing this project, so we should keep it in `peerDependencies`, while also adding it to `devDependencies`.